js/utils.js

```diff
@@ -1596,6 +1596,26 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     });
   }
 
+  function toFormattedString(options) {
+    if (!options) {
```

Comment:
`0` should be returned as `"0"` not `""`.

Add a string type check to the top so that if it's a string the code doesn't need to go through all the other logic